jmx-metrics/src/main/groovy/io/opentelemetry/contrib/jmxmetrics/GroovyRunner.java

```diff
@@ -135,15 +135,6 @@ public void run() {
     for (final Script script : scripts) {
       script.run();
     }
-    flush();
-  }
-
-  public void flush() {
-    groovyMetricEnvironment.flush();
```

Comment:
Looks like `groovyMetricEnvironment` can be converted to a local variable now that it's only used in the constructor.

Reply:
I'm not able to push to this branch so will address in a followup: https://github.com/open-telemetry/opentelemetry-java-contrib/issues/197